snapshots/snapshots-shared/src/main/kotlin/com/emergetools/snapshots/shared/ComposePreviewSnapshotConfig.kt

```diff
@@ -45,13 +47,15 @@ data class ComposePreviewSnapshotConfig(
   /**
    * Stable key name generation that takes into account all diff-relevant variants.
    */
-  fun keyName(): String {
+  fun keyName(shortFqn: Boolean = false): String {
```

Comment:
nit: in general I'm hesitant against boolean params like this since it makes the callsite more confusing. maybe this could be `baseKey: string = originalFqn`?

Reply:
Also test cases would be great for this.